infrastructure/aws/main.tf

```diff
@@ -0,0 +1,155 @@
+data "aws_availability_zones" "available" {}
```

Comment:
Not sure I like the name of this resources just being `available`.

Reply:
don't even really need that block, removed